examples/all-clusters-app/ameba/main/include/ManualDishWasherAlarmCommand.h

```diff
@@ -0,0 +1,172 @@
+    supported.SetField(AlarmMap::kInflowError, 1);     // 0x01, 1
+    supported.SetField(AlarmMap::kDrainError, 1);      // 0x02, 2
+    supported.SetField(AlarmMap::kDoorError, 1);       // 0x04, 4
+    supported.SetField(AlarmMap::kTempTooLow, 1);      // 0x08, 8
+    supported.SetField(AlarmMap::kWaterLevelError, 1); // 0x20, 32
+
+    BitMask<AlarmMap> mask;                       // Set dishwasher alarm mask value
+    mask.SetField(AlarmMap::kInflowError, 1);     // 0x01, 1
+    mask.SetField(AlarmMap::kDrainError, 1);      // 0x02, 2
+    mask.SetField(AlarmMap::kDoorError, 1);       // 0x04, 4
+    mask.SetField(AlarmMap::kTempTooLow, 1);      // 0x08, 8
+    mask.SetField(AlarmMap::kWaterLevelError, 1); // 0x20, 32
+
+    BitMask<AlarmMap> state;                  // Set dishwasher alarm state value
+    state.SetField(AlarmMap::kDrainError, 1); // 0x02, 2
+    state.SetField(AlarmMap::kDoorError, 1);  // 0x04, 4
+    state.SetField(AlarmMap::kTempTooLow, 1); // 0x08, 8
+
+    status = serverInstance.SetSupportedValue(1, supported); // 0x2F, 47
+    if (status != EMBER_ZCL_STATUS_SUCCESS)
+    {
+        err = CHIP_ERROR_INTERNAL;
+        goto exit;
+    }
+
+    status = serverInstance.SetMaskValue(1, mask); // 0x2F, 47
+    if (status != EMBER_ZCL_STATUS_SUCCESS)
+    {
+        err = CHIP_ERROR_INTERNAL;
+        goto exit;
+    }
+
+    status = serverInstance.SetStateValue(1, state); // 0x0E, 14
+    if (status != EMBER_ZCL_STATUS_SUCCESS)
+    {
+        err = CHIP_ERROR_INTERNAL;
+        goto exit;
+    }
+exit:
+    if (err != CHIP_NO_ERROR)
+    {
+        ChipLogError(DeviceLayer, "ManualDishWasherAlarmSetRaiseCommandHandler Failed!\r\n");
+    }
+
+    return err;
+}
+
+CHIP_ERROR ManualDishWasherAlarmSetLowerCommandHandler(int argc, char ** argv)
+{
+    if (argc != 0)
+    {
+        return ManualDishWasherAlarmCommandHelpHandler(argc, argv);
+    }
+
+    CHIP_ERROR err = CHIP_NO_ERROR;
+    EmberAfStatus status;
+    DishwasherAlarmServer & serverInstance = DishwasherAlarmServer::Instance();
+
+    BitMask<AlarmMap> supported;                       // Set dishwasher alarm supported value
+    supported.SetField(AlarmMap::kInflowError, 1);     // 0x01, 1
+    supported.SetField(AlarmMap::kDrainError, 1);      // 0x02, 2
+    supported.SetField(AlarmMap::kDoorError, 1);       // 0x04, 4
+    supported.SetField(AlarmMap::kTempTooLow, 1);      // 0x08, 8
+    supported.SetField(AlarmMap::kWaterLevelError, 1); // 0x20, 32
+
+    BitMask<AlarmMap> mask;                       // Set dishwasher alarm mask value
+    mask.SetField(AlarmMap::kInflowError, 1);     // 0x01, 1
+    mask.SetField(AlarmMap::kDrainError, 1);      // 0x02, 2
+    mask.SetField(AlarmMap::kDoorError, 1);       // 0x04, 4
+    mask.SetField(AlarmMap::kTempTooLow, 1);      // 0x08, 8
+    mask.SetField(AlarmMap::kWaterLevelError, 1); // 0x20, 32
+
+    status = serverInstance.SetSupportedValue(1, supported); // 0x2F, 47
```

Comment:
I would expect the supported and mask values to be set once at cluster init time and then not touched as alarms are raised/lowered....  but that can be a followup.

Reply:
I will update in the future.